cypress/integration/404.spec.ts

```diff
@@ -0,0 +1,27 @@
+it('should display custom 404 page when not found', () => {
+  cy.visit('/not-existing-page', { failOnStatusCode: false })
+
+  cy.title().should('equal', '404 - Not Found | Octoclairvoyant')
+
+  cy.findByRole('heading', {
+    name: 'This page could not be found.',
+  }).should('exist')
+
+  cy.findByText('Octoclairvoyant cannot divine the page you wanted.').should(
+    'exist'
+  )
+
+  cy.findByAltText(/octoclairvoyant reading a crystal ball/i)
```

Comment:
I think we could improve our `alt` text, consider that people who do not see the image, do not know what an "Octoclairvoyant" is.

How about this `alt` text: "A purple octopus reading a crystal ball"?

https://webaim.org/techniques/alttext/

Reply:
Happy to improve that, although it's not an octopus but an octocat 😅

Reply:
Do you mean it's like the GitHub mascot then? They call that an octocat as well? I'm missing the cat like ears then on our mascot???

Reply:
Yeah like the GitHub mascot, but it was complicated to explain it to the designer that made the logo 😂 

I'll keep it simple and just refer to it as "octopus" tho. I'll push the change in a bit.